adr/016-database.md

```diff
@@ -0,0 +1,21 @@
+# 16 Postgres Database
+
+Date: 2024-01-08
+
+## Decision
+
+We will use a Postgres database for storage of project data that needs to persist across restarts.
+
+## Status
+
+Accepted
+
+## Context
+
+Postgres is an open-source, relational, SQL database. It provides a proven and robust feature set in the base implementation as well as add ons if specific features are required
```

Comment:
```suggestion
Postgres is an open-source, relational, SQL database. It provides a proven and robust feature set in the base implementation as well as add ons if specific features are required. Postgres is also well-supported by the community.
```

Reply:
Thanks for adding that! Great suggestion